content/news/2023-07-09-bevy-0.11/index.md

```diff
@@ -0,0 +1,2137 @@
+
+To update an existing Bevy App or Plugin to **Bevy 0.11**, check out our [0.10 to 0.11 Migration Guide](/learn/migration-guides/0.10-0.11/).
+
+Since our last release a few months ago we've added a _ton_ of new features, bug fixes, and quality of life tweaks, but here are some of the highlights:
+
+<!-- more -->
+
+* **Screen Space Ambient Occlusion (SSAO)**: Increase scene render quality by simulating "indirect" diffuse light
+* **Temporal Anti-Aliasing (TAA)**: A popular anti-aliasing technique that blends the current frame with past frames using motion vectors to smooth out artifacts
+* **Morph Targets**: Animate vertex positions on meshes between predefined states. Great for things like character customization!
+* **Robust Constrast Adaptive Sharpening (RCAS)**: Intelligently sharpens renders, which pairs nicely with TAA
+* **WebGPU Support**: Bevy can now render on the web faster and with more features using the modern WebGPU web API
+* **Improved Shader Imports**: Bevy shaders now support granular imports and other new features
+* **Parallax Mapping**: Materials now support an optional depth map, giving flat surfaces a feel of depth through parallaxing the material's textures
+* **Schedule-First ECS APIs**: A simpler and more ergonomic ECS system scheduling API
+* **Immediate Mode Gizmo Rendering**: Easily and efficiently render 2D and 3D shapes for debugging and editor scenarios
+* **ECS Audio APIs**: A more intuitive and idiomatic way to play back audio
+* **UI Borders**: UI nodes can now have configurable borders!
+* **Grid UI Layout**: Bevy UI now supports CSS-style grid layout
+* **UI Performance Improvements**: The UI batching algorithm was changed, yielding significant performance wins
+
+## Screen Space Ambient Occlusion
+
+<div class="release-feature-authors">authors: @JMS55</div>
+
+**Without SSAO**
+![no_ssao](no_ssao.png)
+
+**With SSAO**
+![with_ssao](with_ssao.png)
+
+**SSAO Only**
+![ssao_only](ssao_only.png)
+
+Bevy now supports Screen Space Ambient Occlusion (SSAO). While Bevy already supported shadows from direct lights
+([`DirectionalLight`], [`PointLight`], [`SpotLight`]) via shadow mapping, Bevy now supports shadows from _indirect_ diffuse lighting such as [`AmbientLight`] or [`EnvironmentMapLight`].
+
+These shadows give scenes a more "grounded" feel, by estimating how much surrounding geometry blocks incoming light via the screen-space depth and normal prepasses. You can try it out in the new [SSAO example](https://github.com/bevyengine/bevy/blob/v0.11.0/examples/3d/ssao.rs).
+
+Note that using SSAO with the newly added Temporal Anti-Aliasing leads to a _large_ increase in quality and noise reduction.
```

Comment:
```suggestion
Note that using SSAO with the newly added Temporal Anti-Aliasing leads to a _large_ increase in quality and noise reduction, and is highly recommended.
```

Reply:
I removed this intentionally. I don't think we can "highly recommend" TAA when it is experimental / not working with so many "critical" 3D features like animation.

Reply:
Ah I didn't realize I had said that before and it was removed. Makes sense to me :+1:.